addons/mail/static/src/models/message_list_view_item.js

```diff
@@ -24,7 +24,7 @@ registerModel({
         }),
         notificationMessageView: one('NotificationMessageView', {
             compute() {
-                if (this.message.message_type === 'notification' && this.message.originThread.channel) {
+                if (!this.message.isFromOdooBot && this.message.message_type === 'notification' && this.message.originThread.channel) {
```

Comment:
And I guess same question here too, can't we make the message the proper type and/or subtype instead of adding this hacky condition?